tests/countries/test_indonesia.py

```diff
@@ -303,8 +305,8 @@ def test_labor_day(self):
         name = "Hari Buruh Internasional"
         self.assertHolidayName(name, (f"{year}-05-01" for year in range(1953, 1968)))
         self.assertHolidayName(name, (f"{year}-05-01" for year in range(2014, 2050)))
-        self.assertNoHoliday(
-            f"{year}-05-01" for year in set(range(1968, 2014)).difference({2004, 2008})
+        self.assertNoHolidayName(
```

Comment:
I don't think we should change this. `assertNoHoliday()` is a stricter check here, and check for name's absence in these years is in next line.

Reply:
This might work for `International Labor Day`, but this ran into some issues earlier for `Pancasila Day` test cases as certain Islamic holidays are on these days as well.

Alternatively - since we already got the name checks for year entries, can we simply remove these checks?